src/app/tests/TestDataModelSerialization.cpp

```diff
@@ -162,7 +162,7 @@ void TestDataModelSerialization::TestDataModelSerialization_EncAndDecSimpleStruc
 
         t.a = 20;
         t.b = true;
-        t.c = clusters::TestCluster::SimpleEnum::VALUEA;
+        t.c = clusters::TestCluster::SimpleEnum::SIMPLE_ENUM_VALUE_A;
```

Comment:
`SimpleEnum` already name spaces the value, so repeating the term `SIMPLE_ENUM` in the enum value is not necessary.

Reply:
In xml, we have enum https://github.com/project-chip/connectedhomeip/blob/a9ce3207a39de030ddd86e286696c928d5a5d0d4/src/app/zap-templates/zcl/data-model/chip/wifi-network-diagnostics-cluster.xml#L30

without putting the prefix for those enum, it would complain the compilation error.

20_PERCENTER_DIM_UP_IN_0P5_SECONDS_THEN_FADE_TO_OFF_IN_1_SECOND = 0x00,
    ^
../../zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:1314:5: error: expected identifier
    802__11A  = 0x00,
    ^
../../zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:1315:5: error: expected identifier
    802__11B  = 0x01,
    ^
../../zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:1316:5: error: expected identifier
    802__11G  = 0x02,
    ^

Reply:
Not worth it to fix that at this stage.